index.bs

```diff
@@ -3430,15 +3430,16 @@ engine.
     - Verify that `extraData` is set to the hash of |attToBeSigned| using the hash algorithm employed in "alg".
     - Verify that `attested` contains a `TPMS_CERTIFY_INFO` structure as specified in [[TPMv2-Part2]] section 10.12.3,
         whose `name` field contains a valid Name for |pubArea|,
-        as computed using the algorithm in the `nameAlg` field of |pubArea| using the procedure specified in [[TPMv2-Part1]]
-        section 16.
+        as computed by concatinating algorithm identifier from the signauture algorithm(for RS256 it's sha256) specified in TPM_ALG_ID, with hash of the `pubArea` field as specified in [[TPMv2-Part1]] section 16.
```

Comment:
I'm thinking we do not want to get into this much detail since it is already spec'd in the TPM specs.  Suggest doing only this:
```
as computed using the hash algorithm identified by the value of the 
`nameAlg` field of |pubArea| and using the procedure specified in 
[[TPMv2-Part1]] section 16. 
```

Reply:
@equalsJeffH Navigating TPM specs is a hell. So I think we should be slightly more specific in this example

Reply:
I agree with @equalsJeffH here. Duplicating instructions risks getting them wrong in subtle ways, and I think that risk outweighs the small possible convenience gain.

Reply:
I agree that the TPM specs are difficult to decipher -- i spent a ton of time yesterday trying to figure out how the nameAlg field works, and am not sure my interpretation is correct -- hence not wanting to bake errornous interp into webauthn, as @emlun notes.

Reply:
Also agree - the TPM specs are difficult, so let's not add to confusion by potentially getting it different here.

Reply:
@BradL-Msft Are other changes in this PR OK?